components/user-claim-retriever/src/main/java/com/wso2telco/claims/RemoteClaimsRetriever.java

```diff
@@ -110,7 +113,7 @@ private String getclaimValue(Iterator<String> i, boolean isHashed, Map<String, O
         try {
             Class c = Class.forName(className);
             RemoteClaims remoteClaims = (RemoteClaims) c.newInstance();
-            totalClaims = remoteClaims.getTotalClaims(operatorEndPoint, msisdn);
+            totalClaims = remoteClaims.getTotalClaims(operatorEndPoint, customerInfo);
         } catch (ClassNotFoundException e) {
             log.error(className + " Class Not Found Exception");
```

Comment:
Log and Throw exception. Better to provide a more descriptive error message.

Reply:
fixed